code/game/objects/items/devices/scanners.dm

```diff
@@ -264,147 +271,136 @@ GENE SCANNER
+		var/mutant = humantarget.dna.check_mutation(HULK) \
+			|| targetspecies.mutantlungs != initial(targetspecies.mutantlungs) \
+			|| targetspecies.mutantbrain != initial(targetspecies.mutantbrain) \
+			|| targetspecies.mutantheart != initial(targetspecies.mutantheart) \
+			|| targetspecies.mutanteyes != initial(targetspecies.mutanteyes) \
+			|| targetspecies.mutantears != initial(targetspecies.mutantears) \
+			|| targetspecies.mutanthands != initial(targetspecies.mutanthands) \
+			|| targetspecies.mutanttongue != initial(targetspecies.mutanttongue) \
+			|| targetspecies.mutantliver != initial(targetspecies.mutantliver) \
+			|| targetspecies.mutantstomach != initial(targetspecies.mutantstomach) \
+			|| targetspecies.mutantappendix != initial(targetspecies.mutantappendix) \
+			|| targetspecies.flying_species != initial(targetspecies.flying_species)
+
+		render_list += "<span class='info ml-1'>Species: [targetspecies.name][mutant ? "-derived mutant" : ""]</span>\n"
+		render_list += "<span class='info ml-1'>Core temperature: [round(humantarget.coretemperature-T0C,0.1)] &deg;C ([round(humantarget.coretemperature*1.8-459.67,0.1)] &deg;F)</span>\n"
+	render_list += "<span class='info ml-1'>Body temperature: [round(target.bodytemperature-T0C,0.1)] &deg;C ([round(target.bodytemperature*1.8-459.67,0.1)] &deg;F)</span>\n"
 
 	// Time of death
-	if(M.tod && (M.stat == DEAD || ((HAS_TRAIT(M, TRAIT_FAKEDEATH)) && !advanced)))
-		render_list += "<span class='info ml-1'>Time of Death: [M.tod]</span>\n"
-		var/tdelta = round(world.time - M.timeofdeath)
+	if(target.tod && (target.stat == DEAD || ((HAS_TRAIT(target, TRAIT_FAKEDEATH)) && !advanced)))
+		render_list += "<span class='info ml-1'>Time of Death: [target.tod]</span>\n"
+		var/tdelta = round(world.time - target.timeofdeath)
 		render_list += "<span class='alert ml-1'><b>Subject died [DisplayTimeText(tdelta)] ago.</b></span>\n"
 
 	// Wounds
-	if(iscarbon(M))
-		var/mob/living/carbon/C = M
-		var/list/wounded_parts = C.get_wounded_bodyparts()
+	if(iscarbon(target))
+		var/mob/living/carbon/carbontarget = target
+		var/list/wounded_parts = carbontarget.get_wounded_bodyparts()
 		for(var/i in wounded_parts)
 			var/obj/item/bodypart/wounded_part = i
-			render_list += "<span class='alert ml-1'><b>Warning: Physical trauma[LAZYLEN(wounded_part.wounds) > 1? "s" : ""] detected in [wounded_part.name]</b>"
+			render_list += "<span class='alert ml-1'><b>Physical trauma[LAZYLEN(wounded_part.wounds) > 1? "s" : ""] detected in [wounded_part.name]</b>"
 			for(var/k in wounded_part.wounds)
 				var/datum/wound/W = k
-				render_list += "<div class='ml-2'>Type: [W.name]\nSeverity: [W.severity_text()]\nRecommended Treatment: [W.treat_text]</div>\n" // less lines than in woundscan() so we don't overload people trying to get basic med info
+				render_list += "<div class='ml-2'>[W.name] ([W.severity_text()])</div>" // less lines than in woundscan() so we don't overload people trying to get basic med info
```

Comment:
Bring back the `treat_text`, wound scans honestly don't need to exist and I'd like to remove them someday. The venn diagram of people who know about wound scan mode and don't already know how to solve wounds is very minimal, and switching between them is clunky.